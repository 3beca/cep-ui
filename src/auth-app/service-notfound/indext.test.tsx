import * as React from 'react';
import {render, screen} from '@testing-library/react';
import ServiceNotFoundPage from './index';

test('NotFoundPage snapshot', () => {
    const {container} = render(<ServiceNotFoundPage/>);
    screen.getByLabelText(/authapp no cep service/i);
    expect(container).toMatchSnapshot();
});