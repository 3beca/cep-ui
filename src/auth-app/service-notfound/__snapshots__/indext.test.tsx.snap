// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage snapshot 1`] = `
<div>
  <div
    class="makeStyles-container-1"
  >
    <div
      class="makeStyles-dialog-2"
    >
      <div
        aria-label="authapp no cep service"
      >
        <h3
          class="MuiTypography-root MuiTypography-h3 MuiTypography-colorPrimary"
        >
          CEP Service not found!
        </h3>
      </div>
      <div
        class="makeStyles-message-3"
      >
        <span
          class="MuiTypography-root makeStyles-messageText-4 MuiTypography-caption MuiTypography-colorTextSecondary"
        >
          CEP is not responding in 
          http://localhost:123
          , please visit our docs in order to configure it with a valid url.
        </span>
      </div>
    </div>
  </div>
</div>
`;
